operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-statefulset.yaml

```diff
@@ -57,8 +57,20 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        {{- if (eq $rolloutZone.noDownscale true )}}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        {{- else }}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
+        {{- end }}
       annotations:
         {{- include "mimir.podAnnotations" $args | nindent 8 }}
+         {{- if $rolloutZone.prepareDownscale }}
+        grafana.com/prepare-downscale-http-path: store-gateway/prepare-shutdown
+        grafana.com/prepare-downscale-http-port: {{ include "mimir.serverHttpListenPort" . }}
+        {{- end-}}
+        {{- if $rolloutZone.downscaleLeader }}
+        grafana.com/rollout-downscale-leader: $rolloutZone.downscaleLeader
+        {{- end }}
```

Comment:
these annotations and labels should be set on the stateful set not on the pods themselves. 

that's also what jsonnet does https://github.com/grafana/mimir/blob/b9d91bde56989933280a7587f19f3a23a7fab5dd/operations/mimir/ingester-automated-downscale.libsonnet#L34-L37 

these are the rollouts operator docs https://github.com/grafana/rollout-operator?tab=readme-ov-file#how-scaling-up-and-down-works

Reply:
is this true of the labels as well?